### PR TITLE
SDI renamed from IESD

### PR DESCRIPTION
**Pull request by sungeunbae**
Small change to make qcore aware of the name change from IESD to SDI.
Unfortunately, black made a lot more changes that I actually did.

### CODE REVIEW

File: qcore/geo.py
Comment (jpa198) [inline qcore/geo.py:519]: Wasn't this already in here? Also it's not related to SDI

**Review by jasonmotha — COMMENTED** [review]
No changes requested. But waiting on James's comment  + merge conflict

File: qcore/im.py
Comment (jpa198) [inline qcore/im.py:163]: SDI is displacement, so should have units of m or cm?

File: qcore/geo.py
Comment (jpa198) [reply qcore/geo.py:519]: This will probably be resolved when master is merged in...

File: qcore/im.py
Comment (sungeunbae) [reply qcore/im.py]: Well spotted. Thanks

File: qcore/geo.py
Comment (sungeunbae) [reply qcore/geo.py:519]: Yes, once the PR is merged in (vm_gen_refactor) - it'll be all cleaned up.

File: qcore/im.py
Comment (sungeunbae) [reply qcore/im.py]: unit of cm, it is.

File: qcore/im.py
Comment (jasonmotha) [inline qcore/im.py:165]: Update this comment too. As there are no longer any ratios here.

File: qcore/im.py
Comment (jasonmotha) [inline qcore/im.py:136]: While you are editing this are you able to fix my bug?

`self.component = constants.Components.from_str(self.component)`

File: qcore/im.py
Comment (sungeunbae) [reply qcore/im.py]: Done. Just removed "Ratios are dimensionless" bit.

File: qcore/im.py
Comment (sungeunbae) [reply qcore/im.py]: Done.

**Review by jasonmotha — APPROVED** [review]

**Review by jpa198 — APPROVED** [review]

**Pull request merged**
